m4opt/models/extinction/airmass.py

```diff
@@ -0,0 +1,394 @@
+from ..core import state
+from .core import BaseExtinction
+
+import numpy as np
+import astropy.units as u
+from astropy.coordinates import EarthLocation, AltAz
+from astropy.modeling import custom_model
+from astropy.modeling.models import Const1D, Tabular1D
+from astropy.table import QTable
+from astropy.utils.data import download_file
+
+
+path_to_extin = ('https://raw.githubusercontent.com/astropy/' +
+                 'specreduce-data/main/specreduce_data/' +
+                 'reference_data/extinction/')
```

Comment:
Fun fact: you can concatenate string literals by placing them next to each other without any operator. Just like in C!

```suggestion
path_to_extin = ('https://raw.githubusercontent.com/astropy/'
                 'specreduce-data/main/specreduce_data/'
                 'reference_data/extinction/')
```

Reply:
TMYK (insert rainbow here)